src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -169,6 +169,23 @@ struct TargetStepSize : db::SimpleTag {
   }
 };
 
+struct EndTime : db::SimpleTag {
```

Comment:
You have this tag now in both `InitializationTags` and `Tags` namespaces. You probably only need it once

Reply:
yes, only once; either keep it after initialization, or remove the `create_from_options` from the non-initialization tag